guides/17_Migration_to_3_0.md

```diff
@@ -163,6 +163,117 @@ handle them please remove.
 - StateMachineTransitionFailed
 - AudioDeviceSwitched
 
+## AudioVideo events
+
+We have removed below `AudioVideo` events in v3.
+
+- videoSendHealthDidChange
+- videoSendBandwidthDidChange
+- videoReceiveBandwidthDidChange
+- videoNotReceivingEnoughData
+- estimatedDownlinkBandwidthLessThanRequired
+
+In 3.x you can still replecate these events funtionality through `metricDidReceive`. However `estimatedDownlinkBandwidthLessThanRequired` can not be easily replicated.
```

Comment:
nit: metricsDidReceive.

The example below is self explanatory I think. Also, the last part about `estimatedDownlinkBandwidthLessThanRequired` feels in-complete, so if there is a way suggest checking with Henry if we can add to example or drop it.

Reply:
Both this event and videoNotReceivingEnoughData can not be replicated from the application level cause we do not expose some signal classes in the facade. So I can just say these functions can not be replicated anymore.